test/e2e/Rakefile

```diff
@@ -250,6 +250,27 @@ task :get_latest_configs, [:env] do |task, args|
   end
 end
 
+task :get_docker_logs do
+  puts "\n  >> Getting docker logs into #{LOGS}"
+
+  mk_dir(LOGS)
+  node_logfile = `docker inspect --format='{{.LogPath}}' cardano-node`.gsub("\n",'')
+  wallet_logfile = `docker inspect --format='{{.LogPath}}' cardano-wallet`.gsub("\n",'')
+
+  node_cmd = "sudo cp #{node_logfile} #{LOGS}/node.log"
+  wallet_cmd = "sudo cp #{wallet_logfile} #{LOGS}/wallet.log"
+
+  puts node_cmd
```

Comment:
Is there a way for rake to automatically print commands before they are run?

Reply:
I don't think so. I think I could wrap it in a `cmd` command or something like that.

Reply:
I have added `cmd` method and wrapped all commands into it. (I've found there is a `sh` command that comes with rake, that does the same pretty much, however It was giving me strange failures on Windows for some reason)